configs/esm_software/esm_runscripts/esm_plugins.yaml

```diff
@@ -26,7 +26,15 @@ core:
 
 # To add your custom plugin, see oifs.yaml, section compute_recipe, and compare to the list above.
 # the custom plugin 'preprocess' listed in the compute_recipe in oifs.yaml needs to be installed beforehand with
+#
 # git clone https://github.com/esm-tools-plugins/preprocess
 # cd preprocess
-# if required activate python3 e.g. module load anaconda3/bleeding_edge on mistral
+# if required activate python3 e.g. 
+# module load anaconda3/bleeding_edge on
 # pip install --user 
```

Comment:
You could add that to the list of commands oifs executes when it is installed.

Reply:
Good point, but I'll do that on a later stage given the fact that we are very close to the release

Reply:
OK, I'll make an issue so we don't forget later on